### PR TITLE
update_deps

### PR DESCRIPTION
**Pull request by jreback**

### CODE REVIEW

**Comment by jreback** [comment]
@jakirkham am I supposed to have permissions to cancel (from my fork) on travis? https://travis-ci.org/conda-forge/pandas-gbq-feedstock/pull_requests (not a big deal, just asking)

**Comment by ocefpaf** [comment]
> am I supposed to have permissions to cancel (from my fork) on travis? https://travis-ci.org/conda-forge/pandas-gbq-feedstock/pull_requests (not a big deal, just asking)

Yes, but you may need to sync your Travis-CI account.

**Comment by ocefpaf** [comment]
It looks like some of the new dependencies might not be available for Python 3.6. Let me check that and see if I can fix it.

**Comment by jreback** [comment]
@ocefpaf yeah thanks..

its just the ``google-api-python-client`` (others are ok).

**Comment by ocefpaf** [comment]
> its just the ``google-api-python-client``, its pure-python though, so should be straightforward. (others are ok). 

We never know the deps of the deps, so it may be more onerous than one first expect. But this one was easy and https://github.com/conda-forge/oauth2client-feedstock/pull/2 should fix it. You can restart the CIs here once https://github.com/conda-forge/oauth2client-feedstock/pull/2 is merged.

**Comment by jreback** [comment]
@ocefpaf sure, see my comment though. It looks like it was *already* re-rendered just not merged.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ requirements:
     - python
     - pandas
     - httplib2
-    - google-api-python-client  # [not py36]
+    - google-api-python-client
```
Comment (ocefpaf) [inline recipe/meta.yaml]: :+1:

**Comment by jreback** [comment]
thanks @ocefpaf big travis queue :>

**Comment by ocefpaf** [comment]
@jreback feel free to merge this without waiting for Travis-CI. (I will follow the build on `master` to ensure it passes.)

**Comment by jreback** [comment]
@ocefpaf great thanks!.

**Comment by jreback** [comment]
@ocefpaf hmm, does this need to redo the py3.6 build? (this was just now on osx-64)
```
(pandas) bash-3.2$ conda create -n foo --dry python=3.6 pandas-gbq
Fetching package metadata ...........
Solving package specifications: .

Package plan for installation in environment /Users/jreback/miniconda3/envs/foo:

The following NEW packages will be INSTALLED:

    httplib2:        0.10.3-py36_0      conda-forge
    mkl:             2017.0.1-0         defaults   
    numpy:           1.12.0-py36_0      defaults   
    oauth2client:    2.2.0-py36_0       conda-forge
    openssl:         1.0.2k-1           defaults   
    pandas:          0.19.2-np112py36_1 defaults   
    pandas-gbq:      0.1.3-py36_1       conda-forge
    pip:             9.0.1-py36_1       defaults   
    pyasn1:          0.1.9-py36_0       defaults   
    pyasn1-modules:  0.0.8-py36_0       defaults   
    python:          3.6.0-0            defaults   
    python-dateutil: 2.6.0-py36_0       defaults   
    pytz:            2016.10-py36_0     defaults   
    readline:        6.2-2              defaults   
    rsa:             3.4.2-py36_0       conda-forge
    setuptools:      27.2.0-py36_0      defaults   
    six:             1.10.0-py36_0      defaults   
    sqlite:          3.13.0-0           defaults   
    tk:              8.5.18-0           defaults   
    wheel:           0.29.0-py36_0      defaults   
    xz:              5.2.2-1            defaults   
    zlib:            1.2.8-3            defaults   
```

```
(pandas) bash-3.2$ conda create -n foo --dry python=3.5 pandas-gbq
Fetching package metadata ...........
Solving package specifications: .

Package plan for installation in environment /Users/jreback/miniconda3/envs/foo:

The following NEW packages will be INSTALLED:

    google-api-python-client: 1.6.2-py35_0       conda-forge
    httplib2:                 0.10.3-py35_0      conda-forge
    mkl:                      2017.0.1-0         defaults   
    numpy:                    1.12.0-py35_0      defaults   
    oauth2client:             2.2.0-py35_0       conda-forge
    openssl:                  1.0.2k-1           defaults   
    pandas:                   0.19.2-np112py35_1 defaults   
    pandas-gbq:               0.1.3-py35_1       conda-forge
    pip:                      9.0.1-py35_1       defaults   
    pyasn1:                   0.1.9-py35_0       defaults   
    pyasn1-modules:           0.0.8-py35_0       defaults   
    python:                   3.5.3-1            defaults   
    python-dateutil:          2.6.0-py35_0       defaults   
    pytz:                     2016.10-py35_0     defaults   
    readline:                 6.2-2              defaults   
    rsa:                      3.4.2-py35_0       conda-forge
    setuptools:               27.2.0-py35_0      defaults   
    simplejson:               3.10.0-py35_0      defaults   
    six:                      1.10.0-py35_0      defaults   
    sqlite:                   3.13.0-0           defaults   
    tk:                       8.5.18-0           defaults   
    uritemplate:              3.0.0-py35_0       conda-forge
    wheel:                    0.29.0-py35_0      defaults   
    xz:                       5.2.2-1            defaults   
    zlib:                     1.2.8-3            defaults   

```

**Comment by ocefpaf** [comment]
That is odd. I opened the package and I see 


`meta.yaml`

```yaml
# This file created by conda-build 2.1.4
# meta.yaml template originally from:
# /Users/travis/build/conda-forge/pandas-gbq-feedstock/recipe, last modified Mon Mar  6 17:47:01 2017
# ------------------------------------------------

package:
    name: pandas-gbq
    version: 0.1.3
source:
    fn: pandas-gbq-0.1.3.tar.gz
    sha256: 8974723a4f7847f26eeaf1667f1214037c18a56a4de14180cc8d6fb1bef860d6
    url: https://pypi.io/packages/source/p/pandas-gbq/pandas-gbq-0.1.3.tar.gz
build:
    number: '1'
    script: python setup.py install --single-version-externally-managed --record record.txt
    string: py36_1
requirements:
    build: []
    run:
        - python
        - pandas
        - httplib2
        - oauth2client
test:
    imports:
        - pandas_gbq
about:
    dev_url: https://github.com/pydata/pandas-gbq
    doc_url: http://pandas-gbq.readthedocs.io
    home: https://github.com/pydata/pandas-gbq
    license: BSD 3-clause
    summary: Pandas interface to Google Big Query
extra:
    recipe-maintainers:
        - jreback
        - parthea
```

but I see this in the `meta.yaml.template`.

```yaml
{% set version = "0.1.3" %}
{% set name = "pandas-gbq" %}
{% set sha256 = "8974723a4f7847f26eeaf1667f1214037c18a56a4de14180cc8d6fb1bef860d6" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  fn: {{ name }}-{{ version }}.tar.gz
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: {{ sha256 }}

build:
  number: 1
  script: python setup.py install --single-version-externally-managed --record record.txt

requirements:
  build:
    - python
    - setuptools

  run:
    - python
    - pandas
    - httplib2
    - google-api-python-client # [not py36]
    - oauth2client

test:
  imports:
    - pandas_gbq

about:
  home: https://github.com/pydata/pandas-gbq
  license: BSD 3-clause
  summary: 'Pandas interface to Google Big Query'

  doc_url: http://pandas-gbq.readthedocs.io
  dev_url: https://github.com/pydata/pandas-gbq

extra:
  recipe-maintainers:
    - jreback
    - parthea
```

And the build 1 of package was uploaded 2 days ago accordingly to: https://anaconda.org/conda-forge/pandas-gbq/files

TL;DR
I can't find the culprit right now so I agree. Bump the build number and keep your fingers crossed that a new build will sort this out.

**Comment by ocefpaf** [comment]
Ah! That is probably due the premature upload from when you did not use the fork :smile:

**Comment by jreback** [comment]
ok, so do a new PR and bump build number?

**Comment by ocefpaf** [comment]
> ok, so do a new PR and bump build number?

Yep.

**Comment by jreback** [comment]
done thanks for all of the help!
